fv3core/grid/eta.py

```diff
@@ -0,0 +1,183 @@
+import numpy as np
+
+
+def set_eta(km):
```

Comment:
This is returning quite a few (4) values, which I don't expect from the name+docstring. Can you update this and include type hinting? If you return a NamedTuple and add type hints to the call signature and update the name that should be good, or for returning 4 numpy arrays you can add the outputs to the docstring.